packages/core/src/components/DataTable/index.tsx

```diff
@@ -410,22 +410,19 @@ export class DataTable extends React.Component<DataTableProps & WithStylesProps,
 }
 
 export default withStyles(
-  theme => ({
+  ({ color }) => ({
     table_container: {
       overflowX: 'auto',
     },
     column_header: {
-      borderTop: '1px solid',
-      borderBottom: '1px solid',
-      borderColor: theme!.color.core.neutral[1],
+      borderBottom: `1px solid ${color.core.neutral[1]}`,
```

Comment:
`ui.border` here too. the jump from 1 to 2 isn't that large? and would be consistent with the rest of lunar borders, especially in tables